### ISSUE TITLE
Whether to run a test case filtered out by tags when I explicitly do 

### ISSUE THREAD

**Issue by Eskibear** [microsoft/vscode-java-test#1480]
If a test case is filtered out by tags from settings, it won't run when I choose to run all the cases. It's ok.
However, when I explicit click on the case itself, want to run this single case, currently it still won't run. Should we ignore tag filters when explicitly run a single case?

This case looks stranger if previous status of the case is kept there, see the second half of the recording. 

https://user-images.githubusercontent.com/2351748/187153016-16185df3-eca0-4a7a-94a0-1dbefed4d53c.mp4

**Comment by jdneo** [comment]
Yes, it might surprise the users if they forget that a tag filter has been set.

To solve the problem, I have two ideas:

1. Mark the test item that is triggered but not executed to be `skipped` status.
2. Only allow to filter tags when `Run all tests` is triggered.

BTW, better expose the test configuration might also somehow mitigate the problem.
